docs/content/stable/releases/techadvisories/ta-24313.md

```diff
@@ -0,0 +1,90 @@
+---
+title: TA-24313
+headerTitle: YSQL online index backfill consistency issue
+description: Concurrent index writes from an active workload may be missed when creating an index on a table with existing data.
+headcontent: 10 December 2025
+type: docs
+showRightNav: true
+cascade:
+  unversioned: true
+menu:
+  stable_releases:
+    identifier: ta-24313
+    weight: 1
+rightNav:
+  hideH2: true
+type: docs
+---
+
+|          Product           |  Affected Versions  |  Related Issues   | Fixed In |
+| :------------------------- | :------------------ | :---------------- | :------- |
+| {{<product "ysql">}}       | {{<release "2.14, 2.16, 2.18, 2.20, 2024.1, 2024.2.0">}} | {{<issue 24313>}}, {{<issue 25250>}}, {{<issue 24775>}} | {{<release "2.20.9">}}, {{<release "2024.1.4">}}, {{<release "2024.2.1">}} |
+
+## Description
+
+When creating an index on a table with existing data, concurrent writes from an active workload may be missed. In very specific edge cases, this can result in inconsistent data in the new index.
+
+However, if the concurrent activity on the table during the index backfill only involves appending new rows, this advisory can be safely disregarded.
+
+## Detection
+
+[yb_index_check()](/stable/api/ysql/exprs/func_yb_index_check/) is a utility function that checks if an index is consistent with its base relation and can be used to validate the specific index.
+
+`yb_index_check()` is available in YugabyteDB starting from {{<release "2024.2.3">}}, {{<release "2.20.11">}}, and {{<release "2024.1.6">}}.
+
+If `yb_index_check()` is not available, then a PL/PgSQL function can be installed on any affected YugabyteDB version and used to check all the indexes of a table (or only a specified index) by comparing the expected (table) and actual (index) row counts.
```

Comment:
PL/PgSQL function --> can you please make this a link pointing to https://github.com/yugabyte/yb-tools/blob/main/index-consistency-checker/index-consistency-checker-v1.sql